lib/routes/emails.js

```diff
@@ -31,16 +30,11 @@ module.exports = (log, db, mailer, config, customs, push) => {
         log.begin('Account.RecoveryEmailEnabled', request)
 
         const sessionToken = request.auth.credentials
-        let isEnabled = false
 
         return db.account(sessionToken.uid)
```

Comment:
Shouldn't need the account lookup anymore. Could use 

`return reply({ ok: sessionToken.tokenVerified })`